owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -82,7 +83,11 @@ import timber.log.Timber
  */
 abstract class DrawerActivity : ToolbarActivity() {
 
-    private val drawerViewModel by viewModel<DrawerViewModel>()
+    private val drawerViewModel by viewModel<DrawerViewModel> {
+        parametersOf(
+            account?.name
```

Comment:
What if `account` is null? Did you check where this value is set in `BaseActivity`?

Reply:
In this case, `account` variable cannot be null because is set in `swapToDefaultAccount` method from `BaseActivity` before creating the `DrawerViewModel`. This is the same with other view models like `CapabilityViewModel` or `ShareViewModel`